src/App.js

```diff
@@ -1,26 +1,37 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import Header from './Header';
+import Footer from './Footer';
+import styled, { createGlobalStyle } from 'styled-components';
+import ClassList from './ClassList';
 
-function App() {
+const GlobalStyles = createGlobalStyle`
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;margin: 0;
+  padding: 0;
+  }
+`;
+
+const Grid = styled.div`
+  display: grid;
+  grid-template-rows: 60px auto 60px;
```

Comment:
Why not showing the values in fractions to have it responsive?

Reply:
Will do that later maybe